places-solution-integrations/target/attach-data.md

```diff
@@ -1,76 +1,88 @@
 
 <img src="../../.gitbook/assets/integrations/target/ad-setCondition.png" width="800" /><br />
 
 ## Define the Action
 
-The last thing we need to do is define the `Action`. First, click the "Add" button under the "Actions" section:
+1. Under the **Actions** section, click **Add**.
 
-<img src="../../.gitbook/assets/integrations/target/ad-addAction.png" width="800" /><br />
+2. From the **Extension** drop-down list, select **Mobile Core**.  
 
-Select `Mobile Core` from the "Extension" dropdown and choose `Attach Data` from the "Action Type" dropdown.
+3. From the **Action Type** drop-down list, select **Attach Data**.
 
-In the UI on the right, you have the ability to add a freeform JSON payload that will add data to an SDK event before any extension listening for this event will hear it. In our example, we are adding `mboxparameters` to all requests in this event before the Adobe Target extension process it. This results in each of the requests from this event having the new `mboxparameters` added on the outgoing Target request.
+4. On the right pane, in the **JSON Payload** field, type the data that will be added to this Event.
 
-In the example below, we are adding "poiCity" and "poiName" values to the `mboxparameters` for each request processed in the Target event. The values for the new keys will be determined dynamically by the SDK at the time this event processes:
+5. Click **Keep Changes**.
 
-<img src="../../.gitbook/assets/integrations/target/ad-setAction.png" width="800" /><br />
+On the right pane, you can add a freeform JSON payload that adds data to an SDK event before the extensions listening for this event hear it.
+
+In the following example, **poiCity** and **poiName** values are added to the **mboxparameters** for each request that is processed in the Target event. The values for the new keys are determined dynamically by the SDK at the time this event processes.
 
 {% hint style="info" %}
-This JSON payload uses a special notation for the `"request"` object. In the original event, `"request"` is an array of anonymous objects. When attaching data to all of objects in an array using Attach Data, using the `[*]` notation on a key that is known to contain an array will cause the payload to be applied to all objects within that array.
+This JSON payload uses a special notation for the **request** object. In the original event, **request** is an array of anonymous objects. When attaching data to all of objects in an array using Attach Data, the **[*]** notation on a key that is known to contain an array causes the payload to be applied to all objects in that array.
 
-The notation of `"request[*]"` can be read out loud as "for each object in the `request` array".
+The notation of **request[*]** can be read out loud as _for each object in the **request** array_.
 {% endhint %}
 
-## Save the Rule and Rebuild your Property
+<img src="../../.gitbook/assets/integrations/target/ad-setAction.png" width="800" /><br />
```

Comment:
Please remove this graphic; since there is one in the "Save" section, this one is not necessary.